core/vm/actor/builtin/storage_power/storage_power_actor_state.hpp

```diff
@@ -253,6 +254,10 @@ namespace fc::vm::actor::builtin::storage_power {
 
     outcome::result<Power> getTotalNetworkPower() const;
 
+    outcome::result<Claim> assertHasClaim(const Address &address) const;
```

Comment:
No need to export internal functions to interface.
You can make them as functions in .cpp or at least private.